gson/src/main/java/com/google/gson/util/VersionUtils.java

```diff
@@ -1,49 +0,0 @@
-/*
- * Copyright (C) 2017 The Gson authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.google.gson.util;
-
-/**
- * Utility to check the major Java version of the current JVM.
- */
-public class VersionUtils {
```

Comment:
Ok, it is meant to be internal. I will move it out.

Reply:
It's not good that our process allowed this kind of mistake.

I propose some process changes:
 - Code isn't merged without a code review. If the code review is stalled the change is stalled. Err on the side of not merging.
 - Releases aren't released without a heads up to other maintainers. That gives us another opportunity to avoid mistakes.

Reply:
@swankjesse @JakeWharton  Heads up to other maintainers is a good idea. How about a 2 day wait for response? 

Code merging can't keep waiting for reviewers who are not responding. Let's set some SLA on it too.

Reply:
Let's plan a mailing list (or someother mechanism) that will alert all maintainers, and they will be expected to respond in a timely manner. 
Given that this PR breaks (theoretical) binary compatibility, let's get the next release out ASAP. I plan to publish 2.8.5 with this change. Any concerns?

Reply:
I don't understand why this class was moved to create the binary incompatible change. Releasing this seems like a mistake.

Reply:
I made the mistake in Gson 2.8.4 in releasing a class in package com.google.gson.utils which seems like an official package. However, the term "official" is just arbitrary and applies to the package named "internal". Our intention was never to make VersionUtils public and likely no one has used it given that 2.8.4 has been out for a very short time. 
In Gson 2.8.5, we will undo this mistake and take the risk of breaking code which happen to rely on that VersionUtil class. Likely there will be no such person. But we need to publish Gson 2.8.5 quickly, otherwise the chance of someone using that package keeps increasing.

Reply:
I see. I don't have a particularly strong opinion about it then, except that it seems to lend credibility to Jesse's suggestion of not merging anything or making releases without review. Given the scope of this project that's dangerous behavior.

I don't see any reason for an SLA. Gson doesn't need to be moving very fast.